notebooks/odm2api_sample_fromsqlite.ipynb

```diff
@@ -0,0 +1,418 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# odm2api sample\n",
+    "Largely from https://github.com/ODM2/ODM2PythonAPI/blob/master/Examples/Sample.py   \n",
+    "- 2/7/2016. Tested successfully with `sfgeometry_em_1` branch, with my overhauls. Using `odm2api_dev` env.\n",
+    "- 2/1 - 1/31. Errors with SamplingFeatures code, with latest odm2api from master (on env `odm2api_jan31test`). *The code also fails the same way with the `odm2api` env, but it does still run fine with the `odm2api_jan21` env! I'm investigating the differences between those two envs.*\n",
+    "- 1/22-20,9/2016.\n",
+    "\n",
+    "Emilio Mayorga"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import sys\n",
+    "import os\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import dates"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from odm2api.ODMconnection import dbconnection\n",
+    "from odm2api.ODM2.services.readService import *"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create a connection to the ODM2 database\n",
+    "# ----------------------------------------\n",
+    "odm2db_fpth = 'ODM2-LittleBear1.sqlite'\n",
```

Comment:
*I'm pasting here the email I sent you on Friday:*

The odm2api demo notebook is here:
https://github.com/emiliom/stuff/blob/master/odm2api_sample_fromsqlite.ipynb

But note that it uses a SQLite file from my local drive, with a hardwired file path. I've uploaded that SQLite file (3.3 MB) into a repo of mine here:
https://github.com/emiliom/stuff/blob/master/ODM2-LittleBear1.sqlite

Feel free to grab both of those files and do whatever you need to do to create a binder live notebook demo. That'll be fun to see, thanks for offering to do that!